RELEASES.md

```diff
@@ -1,3 +1,100 @@
+Version 1.42.0 (2020-03-12)
+==========================
+
+Language
+--------
+- [You can now use the slice pattern syntax with subslices.][67712] e.g.
```

Comment:
Should this just be "with slices"? I'm not sure subslices are a thing?

Reply:
See #67712 -- I think "with slices" doesn't convey what's new, which is using `..` to match a subset of the slice.

Reply:
Ah. Interesting. Okay, then this seems fine.

Reply:
Subslices are a thing . :)